jsAgent/links.js

```
@@ -1,45 +1,125 @@
+    putContent: function(links, linkFontColor, strokeColor){
+        let temp_content_html=`
+            <div id="bw-links-content" style="margin: 0;padding: 26px 34px 44px; background: white;">
+                <ul >
+                {{links}} 
+                </ul>
+            </div>
+        `;
+        let li_html="";
+        for (let i = 0; i < links.length; i++) {
+            const link = links[i];
+            let content_link = global_content_html.replace(new RegExp('{{link}}', 'g'), link.url );
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)